### PR TITLE
Release announcement for scales 1.1.0

### PR DESCRIPTION
**Pull request by hadley**
TODO:

* [x] update date

### CODE REVIEW

File: content/blog/2019-11-scales-1-1-0/index.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  fig.width = 5, 
+  fig.align = 'center',
+  fig.asp = 0.25, # 1 / phi
+  out.width = "700px",
+  fig.path = "figs/"
+)
+```
+
+We're thrilled to announce the release of [scales](https://scales.r-lib.org/) 1.1.0.
+The scales package provides much of infrastructure the underlies ggplot2's scales, and using it allow you to customize the transformations, breaks, and labels used by ggplot2. Get the latest version with:
```
Comment (batpigandme) [inline content/blog/2019-11-scales-1-1-0/index.Rmarkdown]:
```suggestion
The scales package provides much of the infrastructure the underlies ggplot2's scales, and using it allow you to customize the transformations, breaks, and labels used by ggplot2. Get the latest version with:
```

File: content/blog/2019-11-scales-1-1-0/index.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+```{r, eval = FALSE}
+install.packages("scales")
+```
+
+The biggest improvements in this release of scales are related to usability and documentation:
```
Comment (batpigandme) [inline content/blog/2019-11-scales-1-1-0/index.Rmarkdown]:
```suggestion
The biggest improvements in this release are related to usability and documentation:
```

File: content/blog/2019-11-scales-1-1-0/index.Rmarkdown [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+* Axis breaks and labels have a new naming scheme: functions that generate
+  breaks from limits are called `breaks_`; functions that generate labels
+  from breaks are called `labels_`.
+
+* All the examples for breaks and labels have been overhauled to use new 
+  `demo_continuous()`, `demo_discrete()`, and `demo_log10()`. These make
+  it much easier to see how you'd use scales functions with ggplot2, so
+  when you're [browsing the documentation](https://scales.r-lib.org/reference/index.html),
+  you can immediately how the functions work with ggplot2.
```
Comment (batpigandme) [inline content/blog/2019-11-scales-1-1-0/index.Rmarkdown]:
```suggestion
  you can immediately see how the functions work with ggplot2. 
```

File: content/blog/2019-11-scales-1-1-0/index.Rmarkdown [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    creates labels for a date axis that only show the components of the date 
+    that have changed since the previous label.
+
+    ```{r}
+    five_months <- as.POSIXct(c("2019-11-01", "2020-03-01"))
+    demo_datetime(five_months)
+    demo_datetime(five_months, labels = label_date_short())
+    ```
+
+See the [change log](https://scales.r-lib.org/news/index.html#scales-1-1-0) for a full list of all changes and bug fixes in this version.
```
Comment (batpigandme) [inline content/blog/2019-11-scales-1-1-0/index.Rmarkdown]:
```suggestion
See the [change log](https://scales.r-lib.org/news/index.html#scales-1-1-0) for a full list of changes and bug fixes in this version.
```

File: content/blog/2019-11-scales-1-1-0/index.Rmarkdown (hunk 1 above)
Comment (thomasp85) [reply content/blog/2019-11-scales-1-1-0/index.Rmarkdown]: > much of the infrastructure *that* underlies

Don't know how to suggest changes to suggestions

**Pull request merged**